packages/core/upload/admin/src/components/TableList/CellContent.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+import { getFileExtension } from '@strapi/helper-plugin';
+import { Typography } from '@strapi/design-system/Typography';
+
+import { PreviewCell } from './PreviewCell';
+import { formatBytes } from '../../utils';
+
+export const CellContent = ({
+  alternativeText,
+  content,
+  cellType,
+  elementType,
+  mime,
+  fileExtension,
+  thumbnailURL,
+  url,
+}) => {
+  const { formatDate } = useIntl();
+  switch (cellType) {
+    case 'image':
+      return (
+        <PreviewCell
+          alternativeText={alternativeText}
+          fileExtension={fileExtension}
+          mime={mime}
+          type={elementType}
+          thumbnailURL={thumbnailURL}
+          url={url}
+        />
+      );
+    case 'date':
+      return <Typography>{formatDate(content)}</Typography>;
+
+    case 'size':
+      if (elementType === 'folder') return <Typography>-</Typography>;
+
+      return <Typography>{formatBytes(content)}</Typography>;
+
+    case 'ext':
+      if (elementType === 'folder') return <Typography>-</Typography>;
+
+      return <Typography>{getFileExtension(content).toUpperCase()}</Typography>;
+
+    case 'text':
+      return <Typography>{content}</Typography>;
+
+    default:
+      return <Typography>-</Typography>;
```

Comment:
Just as a suggestion: I'm not sure if this is a good a11y practice (same for the CM) and I'm wondering if it would be better if we would render an aria-label here saying, this field is empty or defining aria-hidden on the value?

Opinions, @joshuaellis ?

Reply:
I like the idea of an aria-label!

Reply:
added a translated aria label [here](https://github.com/strapi/strapi/pull/15085/commits/dc055b43710f3606b5f450f318f99638d50ae00e)

Reply:
i like the label.